helm/cluster-observability-operator/README.md

```diff
@@ -0,0 +1,171 @@
+This component has been copied with few adapations from the [lls-observability](https://github.com/rh-ai-quickstart/lls-observability).
+
+# Cluster Observability Operator Helm Chart
+
+This Helm chart installs the OpenShift Cluster Observability Operator on OpenShift clusters using the Operator Lifecycle Manager (OLM).
+
+## Overview
+
+The Cluster Observability Operator provides comprehensive cluster-wide observability capabilities, including:
+- Cluster-wide telemetry collection and management
+- Integration with OpenShift monitoring stack
+- Custom resource management for observability configuration
+- Enhanced cluster visibility and monitoring
+
+## Prerequisites
+
+- OpenShift 4.x cluster
+- Cluster administrator privileges
+- Helm 3.x installed
+
+## Installation
+
+### Basic Installation
+
+```bash
+# Install the operator
+helm install cluster-observability-operator ./cluster-observability-operator
+```
+
+### Installation with Custom Values
+
+```bash
+# Install with custom namespace and configuration
+helm install cluster-observability-operator ./cluster-observability-operator \
+  --set namespace.name=my-cluster-obs-namespace \
+  --set subscription.channel=stable
+```
+## Configuration
+
+### Default Values
+
+The chart comes with sensible defaults for OpenShift environments:
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| `namespace.name` | Namespace for the operator | `openshift-cluster-observability-operator` |
+| `namespace.create` | Whether to create the namespace | `true` |
+| `subscription.channel` | Operator subscription channel | `stable` |
+| `subscription.packageName` | Operator package name | `cluster-observability-operator` |
+| `subscription.source` | Catalog source | `redhat-operators` |
+| `subscription.installPlanApproval` | Install plan approval mode | `Automatic` |
+| `subscription.startingCSV` | Starting CSV version | `cluster-observability-operator.v1.2.0` |
+| `operatorGroup.targetNamespaces` | Target namespaces for operator scope | `[]` (cluster-wide) |
+
+### OperatorGroup Configuration
+
+The Cluster Observability Operator is configured for cluster-wide deployment (AllNamespaces install mode) to enable comprehensive cluster monitoring and observability. The OperatorGroup uses an empty `targetNamespaces` array to achieve this scope while keeping the operator installed in its dedicated namespace.
```

Comment:
We should probably check with Mikhail to make sure he does not have any concerns about cluster wide installation.